velox/docs/monthly-updates/december-2022.rst

```diff
@@ -0,0 +1,37 @@
+********************
+December 2022 Update
+********************
+
+Presto Functions
+================
+
+* Add :func:`array_has_duplicates`, :func:`crc32`, :func:`hmac_sha1`,
+  :func:`hmac_sha256`, :func:`hmac_sha512`, :func:`json_size` scalar functions.
+
+Spark Functions
+===============
+
+* Add shiftleft and shiftright scalar functions.
+
+Substrait
+=========
+
+* Add support for OrderBy/TopN/Limit plan nodes.
+
+Performance and Correctness
+===========================
+
+* Add nightly job to run Expression Fuzzer using Spark SQL functions.
+* Add nightly job to run Join Fuzzer.
+* Add basic support for window queries to Aggregation Fuzzer.
+* Add support for CAST and SWITCH special forms to Expression Fuzzer.
+
+Credits
+=======
+
+Abhash Jain, Aditi Pandit, Arpit Porwal, Austin Dickey, Bikramjeet Vig, Ge Gao,
+Huameng Jiang, Jacob Wujciak-Jens, Jimmy Lu, Karteek Murthy Samba Murthy,
```

Comment:
Can we also add Raul cumplido and Sasha Krassovsky to this ?

Reply:
This list is auto-generated using a command like this:

```
git log --since='Dec 1 2022' --until='Dec 31 2022' --pretty=format:"%an" | grep -v svcscm | sort | uniq | tr '\n' ','
```

Reply:
Not sure why Sasha's name doesnt come up : https://github.com/facebookincubator/velox/pull/3377 .

Reply:
I see Sasha Krassovsky in the list. Do you not?

Reply:
Apologies, I seemed to have missed it.